benchto-driver/src/main/java/com/teradata/benchto/driver/loader/BenchmarkLoader.java

```diff
@@ -56,7 +56,6 @@
 import static com.teradata.benchto.driver.loader.BenchmarkDescriptor.VARIABLES_KEY;
```

Comment:
> No tests needed to be changed? Maybe add sample test like the one in commit message.

That would only the purpose of cementing the design decision, without giving more coverage (function-wise).